apps/vessel-history/features/vessels/sources/gfw.slice.ts

```diff
@@ -23,40 +37,18 @@ export const toVessel: (data: GFWDetail) => VesselWithHistory = (data: GFWDetail
     vesselType: data.vesselType,
     authorizations: [],
     history: {
-      callsign: {
-        byCount: data.otherCallsigns,
-        byDate: [],
-      },
-      geartype: {
-        byCount: [],
-        byDate: [],
-      },
-      imo: {
-        byCount: data.otherImos,
-        byDate: [],
-      },
-      shipname: {
-        byCount: data.otherShipnames,
-        byDate: [],
-      },
-      mmsi: emptyHistory,
-      owner: emptyHistory,
-      flag: emptyHistory,
-      depth: emptyHistory,
-      length: emptyHistory,
-      grossTonnage: emptyHistory,
-      vesselType: {
-        byCount: [],
-        byDate: [
-          {
-            value: data.vesselType,
-            endDate: data.lastTransmissionDate,
-            firstSeen: data.firstTransmissionDate,
-            source: VesselAPISource.GFW,
-          },
-        ],
-      },
-      operator: emptyHistory,
+      callsign: getHistoryField(data, 'callsign', data.otherCallsigns),
+      geartype: getHistoryField(data, 'geartype'),
+      imo: getHistoryField(data, 'imo'),
```

Comment:
`data.otherImos` should be assigned to the `byCount` arg, right?

Reply:
yes